server/data/mappers/prePrisonQualificationsMapper.ts

```diff
@@ -0,0 +1,16 @@
+import type { CiagInduction } from 'ciagInductionApiClient'
+import type { PrePrisonQualifications } from 'viewModels'
+
+const toPrePrisonQualifications = (ciagInduction: CiagInduction): PrePrisonQualifications => {
+  if (!ciagInduction || !ciagInduction.qualificationsAndTraining) {
+    return { problemRetrievingData: false, highestEducationLevel: undefined, additionalTraining: undefined }
+  }
+
+  return {
+    problemRetrievingData: false,
+    highestEducationLevel: ciagInduction.qualificationsAndTraining.educationLevel,
+    additionalTraining: ciagInduction.qualificationsAndTraining.additionalTraining,
```

Comment:
I'm not sure about this - I think there's an array involved isn't there?

Reply:
Good point, I'll change this.